.github/workflows/production_infrastructure.yml

```diff
@@ -1522,8 +1522,8 @@ jobs:
               DTFS_PORTAL_SCHEDULER="${{ secrets.DTFS_PORTAL_SCHEDULER }}" \
               FETCH_WORKFLOW_TYPE_B_SCHEDULE="${{ secrets.FETCH_WORKFLOW_TYPE_B_SCHEDULE }}" \
               EXTERNAL_API_URL="http://$EXTERNAL_API_URL" \
-              DTFS_CENTRAL_API="http://$dtfs_central_api_url" \
-              TFM_API="http://$tfm_api_url" \
+              DTFS_CENTRAL_API_URL="http://$dtfs_central_api_url" \
```

Comment:
do we need to add the `DTFS_` part? conscious it's not consistent with the other microservices

Reply:
It is because the microservice has `dtfs_` prefixed.